spec/core.md

```diff
@@ -18,187 +18,253 @@ DID, after the prefix, is the case-sensitive [[ref: method-specific identifier]]
-untrusted server of content. It is the combination of target systems and a few
+untrusted server of content. It is the combination of target systems and some
 KERI mechanisms, _together_, that constitutes this method's verifiable data
-registry.
+registry. In short, verifying the DIDDoc by processing the KEL using KERI puts
+the "s" in `did:webs`.
 
 ### Method-Specific Identifier
 
-There are two forms of [[ref: method-specific identifier]] ([[ref: MSI]]) for
-`did:webs`. One is _resolvable_; the other is _stable_. The resolvable form is
-more common and intuitive.
-
-### Resolvable Form
-
-A [[ref: resolvable]] MSI is constructed by transforming an input called the
-[[ref: DID doc URL]] ([[ref: DDURL]]). This MUST be an HTTPS URL that references
-a DID doc (either directly, or ultimately, via redirects). The transformation to
-MSI must be reversible back to DDURL, because resolving the DID requires clients
-to fetch DID docs from the MSI.
-
-The DDURL MUST NOT include a query string (although it could redirect to a URL
-that contains one). The DDURL MUST NOT include relative path components ("./" or
-"../"). The DDURL MUST NOT include multiple consecutive slashes once the path
-portion begins. The DDURL MUST NOT include a fragment OR redirect to a URL that
-includes a fragment. It MUST return a resource that the web server identifies in
-headers as IANA media type `application/json`. The DDURL and its corresponding
-X509 certificate(s) also have some security requirements described
-[elsewhere](#security_considerations).
-
-The transformation begins by removing the "https://" prefix from this URL. If
-the path portion of the URL consists exactly of "/.well-known/did.json", this
-suffix is also removed. If the URL ends in a slash, the final slash is removed.
-All remaining slashes in the URL are replaced with colons. If the URL ends with
-".json", this suffix is deleted. If the URL includes a port, the colon that
-delimits the port number MUST be percent-encoded so colons in the MSI map only
-to slashes.
-
+The `did:webs` method specific identifier should be considered to have two
+parts, a fully qualified domain name with an optional path (identical to
+`did:web`), plus a KERI AID (autonomic identifier) that is always the final
+component of the path. A KERI AID is a unique-identifier derived from the
+inception event of a KERI identifier--the first item in the [[ref: KEL]]. Any
+KERI identifier (DIDs or AIDs) that reference the same AID must, by definition,
+refer to the same KEL, and hence, be the same identifier. An `did:webs` DIDs
+that have the same AID are by definition, synonyms of one another.
+
+The fully qualified domain name is secured by a TLS/SSL certificate with an
+optional path. The formal rules describing valid domain name syntax are
+described in [RFC1035], [RFC1123], and [RFC2181]. The fully qualified domain
+name MUST match the common name used in the SSL/TLS certificate, and MUST NOT
```

Comment:
I know this is copied from did:web, but it's wrong. There's no reason it can't include an IP address, and no reason for the common name requirement other than the mistaken idea that the DID identifies the website.

With did:web, the reason to require HTTPS is so you can know who controls the DID. With did:webs, the reason to require HTTPS is explicitly NOT to know who controls the DID -- the whois folder can tell you that -- but to guarantee that resolution isn't eavesdropped. You can use any old self-signed cert for that. You simply want to guarantee that the cert and the host are consistent, and the cert is not rejected in a way that would undermine confidentiality -- nothing more.